docs/install/install-serving-with-yaml.md

```diff
@@ -34,6 +34,15 @@ To install the serving component:
 
 ## Install a networking layer
 
+The networking layer in Knative Serving routes incoming and outgoing traffic for your Knative installation.
+Knative enables you to choose a networking layer and then converts Knative’s resources
+into the specific configuration for your chosen networking solution.
```

Comment:
This is accurate, but I might say "translates" rather than "converts"; I'm not sure I have a particular rationale for that word choice, except that the original Knative resources still exist.